libraries/Microsoft.Bot.Builder.Dialogs/SkillDialog.cs

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.Bot.Builder.Dialogs
     public class SkillDialog : Dialog
     {
         private const string DeliverModeStateKey = "deliverymode";
+        private const string SkillConversationIdStateKey = "Microsoft.Bot.Builder.Dialogs.SkillDialog.SkillConversationId";
```

Comment:
>Microsoft.Bot.Builder.Dialogs.SkillDialog.SkillConversationId [](start = 60, length = 61)

hmmm...maybe _ instead of period?  Although maybe the fact this is not accessible from expressions is a feature and not a bug.